Open training/src/Topic_0/Hospital.java

```diff
@@ -0,0 +1,118 @@
+package Topic_0;
+
+//3) Design a hospital and provide classes to build it (builder, factories, etc).
+
+// We'll assume "hospital" refers to the level of "luxury" it has,
+//(how many doctors and nurses does it have? what type of rooms are available?)
+//and not the construction of the building itself.
+
+public class Hospital {
+
+	private int floors; // how many floors does the hospital have?
+	private int doctors; // how many doctors?
+	private int nurses; // how many nurses?
+	private int assistants; // how many helpers (janitors, secretary, etc) does
+							// it have?
+	private Boolean operation_theater; // does it have an operation theater?
```

Comment:
You could use the primitive type instead of Boolean, like you did before when using int instead of Integer